charts/datadog/templates/_containers-common-env.yaml

```diff
@@ -151,3 +151,13 @@ Return a list of env-vars if the cluster-agent is enabled
         key: token
 {{- end }}
 {{- end -}}
+
+
+{{- define "kpi-envvar" -}}
```

Comment:
Isn't this redundant since env vars are inlined in both trace agent and DCA containers?

Reply:
Thanks for catching it. This code re-appeared after merge from remote branch.